patch/ros-humble-steering-controllers-library.patch

```diff
@@ -10,3 +10,15 @@ index 095db7674b..88063ea4a2 100644
  #include <cmath>
  #include <memory>
  #include <queue>
+diff --git a/steering_controllers_library/src/steering_odometry.cpp b/steering_controllers_library/src/steering_odometry.cpp
+index 3e1b586807..dbbbf6f850 100644
+--- a/steering_controllers_library/src/steering_odometry.cpp
++++ b/steering_controllers_library/src/steering_odometry.cpp
+@@ -19,6 +19,7 @@
+ 
+ #include "steering_controllers_library/steering_odometry.hpp"
+ 
++#define _USE_MATH_DEFINES
```

Comment:
I am afraid this will not work, as `#include <cmath>` is included earlier (line 10). A safer option is to define it via CMake, see https://github.com/ros-controls/ros2_controllers/pull/1036/files .

Reply:
Actually I got confused, the earlier line 10 is part of another header. Anyhow, defining the macro via CMake ensure that is not possible for cmath to be included without the macro defined.

Reply:
Thank you :)